### ISSUE TITLE
Lesson Feedback

### ISSUE THREAD

**Issue by KatieMFritz** [KatieMFritz/codelab-sinatra-helper-methods#1]
@chrisvfritz: 
## Page 1

I was actually wondering how to do that just the other day! Added it to my business site.
## Page 5

![sinatra-helper-methods-1](https://cloud.githubusercontent.com/assets/7085184/11227968/8447395c-8d57-11e5-9a72-99081b9a5013.png)
In this example (page 5) do you need to include current_time = Time.now for the second method too? Or does it carry over from the first method? If it's not a typo, explain why.

Also on the page 5 example, I don't get how the One Method To Rule Them All knows what current_date_usa and current_date_europe are. If it's getting it from the code snippet above, then it's really three methods, isn't it? This doesn't seem more efficient to me.
## Page 6

On page 6, good idea to say where pipe can be found. I would rephrase it though: "... double pipes (||). Pipe ( | ) can usually be found on a key above the Enter key." Juuuust to be clear that you won't find || on a keyboard.

I love the happy && you know it example. :laughing:
## Page 7
- It's amirite, not amiright. Get your slang rite.
- I would not know to google "ruby compare case insensitive." I'd probably start with "ruby don't pay attention to case" or "Ruby uppercase and lowercase doesn't matter" or something else clumsy, and refine based on the vocabulary in the results I see.
## Page 8

> Well, we've already created strings with "" and ''. When creating multiline strings though, it's often useful to use a percent (%) with any opening and closing symbol, like the parentheses. 

Why is it useful? Is this another alternative to "" in Ruby, and it works better than "" or '' when you have larger, multiline strings? Or is unnecessary but makes reading the code more convenient?

`<%= sidebar_widget("This is my title", "This is my content.") %>`
Whoa. Whoa. I think I get how WordPress functions work now. 

Nice. I bet I can use this for all my bootstrap divs on my site too.

**Comment by KatieMFritz** [comment]
Also on the page 5 example, I don't get how the One Method To Rule Them All knows what current_date_usa and current_date_europe are. If it's getting it from the code snippet above, then it's really three methods, isn't it? This doesn't seem more efficient to me.

**Comment by KatieMFritz** [comment]
On page 6, good idea to say where pipe can be found. I would rephrase it though: "... double pipes (||). Pipe ( | ) can usually be found on a key above the Enter key." Juuuust to be clear that you won't find || on a keyboard.

**Comment by KatieMFritz** [comment]
@chrisvfritz I love the happy && you know it example. :laughing:

**Comment by KatieMFritz** [comment]
@chrisvfritz 
## Page 7:
- It's amirite, not amiright. Get your slang rite.
- I would not know to google "ruby compare case insensitive." I'd probably start with "ruby don't pay attention to case" or "Ruby uppercase and lowercase doesn't matter" or something else clumsy, and refine based on the vocabulary in the results I see.

**Comment by chrisvfritz** [comment]
All addressed!
